src/ApiManagement/ApiManagement/help/New-AzApiManagementCertificate.md

```diff
@@ -95,6 +97,21 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -KeyVault
```

Comment:
Can you add an example that demonstrate the use of `-KeyVault`?

Reply:
Alternatively, add "See `New-AzApiManagementKeyVaultObject` for details." to parameter description